stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -144,6 +149,15 @@ class FPDF:
         format: _Format | tuple[float, float] = ...,
         font_cache_dir: Literal["DEPRECATED"] = ...,
     ) -> None: ...
+    # The following definition crashes stubtest.
+    # def set_encryption(
+    #    self,
+    #    owner_password: str,
+    #    user_password: str | None = None,
+    #    encryption_method: EncryptionMethod | str = ...,
+    #    permissions: AccessPermission = ...,
+    #    encrypt_metadata: bool = False,
+    # ) -> None: ...
```

Comment:
This needs further investigation for which I didn't have time yet.

Reply:
I just ran this against stubtest HEAD and this seems to fix it. I'll update the comment.